File: tests/expectations/tag/union.c
#include <stdint.h>
#include <stdlib.h>
#include <stdbool.h>

struct Opaque;

union Normal {
  int32_t x;
  float y;
};

union NormalWithZST {
  int32_t x;
  float y;
};

void root(struct Opaque *a, union Normal b, union NormalWithZST c);

File: tests/expectations/union.zig
const std = @import("std");

const Opaque = opaque {};

pub const Normal = extern union {
    x: i32,
    y: f32,
};

pub const NormalWithZST = extern union {
    x: i32,
    y: f32,
};

pub extern fn root(a: ?*Opaque, b: Normal, c: NormalWithZST) anyopaque;
